TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/human/HumanActivity.java

```diff
@@ -66,31 +67,32 @@ public void onDirectionReleased(Direction direction) {
 
         @Override
         public void onLazersFired() {
-            commandRepeater.startRepeatingCommand(LAZERS);
+            // no-op for debug
```

Comment:
Removed the command because we are only sending `Direction` which this is not a part of.